src/Npgsql/NpgsqlWriteBuffer.cs

```diff
@@ -164,7 +162,9 @@ internal async Task DirectWrite(byte[] buffer, int offset, int count, bool async
             }
         }
 
-#if !NETSTANDARD2_0 && !NET461
+        internal Task DirectWrite(byte[] buffer, int offset, int count, bool async)
```

Comment:
Let's remove this. At the call site we should call the appropriate Span/Memory overload (based on the async flag).

Reply:
Agreed!